scala/build/Dockerfile.latest

```
@@ -0,0 +1,87 @@
+    postgresql-client default-jre default-jdk apt-transport-https ca-certificates \
+    software-properties-common chromium-bsu chromium-browser xvfb python3-full \
+    make build-essential 
+
+# Create a virtual environment
+RUN python3 -m venv /venv
+
+# Activate the virtual environment
+ENV PATH="/venv/bin:$PATH"
+
+# Upgrade pip within the virtual environment
+RUN pip install --upgrade pip
```

Hadolint

[warning] Avoid use of cache directory with pip. Use `pip install --no-cache-dir <package>`

(DL3042)